src/components/afm/DataSourceProvider.tsx

```diff
@@ -25,6 +25,7 @@ export interface IDataSourceProviderProps {
 }
 
 export interface IDataSourceProviderInjectedProps {
+    exportTitle?: string;
```

Comment:
where it is used?

Reply:
exportTitle (name of exported file) will be used inside VisualizationLoadingHOC.tsx.
- It can be the name of chart type (passed from DataSourceProvider)
- Or Insight's name (passed from uri/Visualization)